### DISCUSSION TITLE
"Extensions have been modified on disk. Please reload the window." How to know which extension has been modified?

### DISCUSSION THREAD

**Discussion by KrabsWong** [microsoft/vscode-discussions#82] [Extension Development QnA] [answered]
I got a notification said "Extensions have been modified on disk. Please reload the window." when I open my VS Code.
How to know, which extensions have been modified? and what were modified?

Is there any log or command to show more detail information for this notification?
<img width="493" alt="image" src="https://user-images.githubusercontent.com/12368943/181915472-2ded01fb-8f78-4b2d-a43f-a837f2614711.png">

**Comment by alefragnani** [comment, marked as answer]
Hi @KrabsWong ,

Right now, you can't. There is no additional information in the `Console` nor any command to display such info.

I suggest you to open a feature request in VS Code repo (https://github.com/microsoft/vscode/issues/new?assignees=&labels=&template=feature_request.md), and ask for such details.

I'm not sure how an extension has been _modified_ in your case, but I've never seen such message. I was able to reproduce the behavior when I intentionally updated the `package.json` file from some random extension I have installed, but not its source. I didn't dig in VS Code source to verify that, but that message is here if you would like to take a look: https://github.com/microsoft/vscode/blob/b91245de85dc5429ad340fa491abaa4510da4ec4/src/vs/workbench/services/extensions/electron-sandbox/cachedExtensionScanner.ts#L68

Hope this helps
